extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -215,8 +215,7 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
           '&adx=-?[0-9]+&ady=-?[0-9]+&u_aw=[0-9]+&u_ah=[0-9]+&u_cd=24' +
```

Comment:
Let's add a test where height and/or width are 'auto' and experiment is on.  What is the expected result?